connectors/connectors.sh

```diff
@@ -85,6 +101,16 @@ fi
 update_connector "bigquery" "$BIGQUERY_CONNECTOR_VERSION"
 update_connector "gcs" "$GCS_CONNECTOR_VERSION"
 
+if [[ $UPDATED_GCS_CONNECTOR != true ]]; then
```

Comment:
It is better to quote it as "true", otherwise it might be considered as a command.

Reply:
It's not necessary in `[[` braces.